specs/Deployment/DeploymentAPI.md

```diff
@@ -106,25 +122,27 @@ concern to some developers. To address these concerns, developers can instead ca
 and verify that the required packages are missing. If so, then call Initialize() on another thread
 to handle the time delay more elegantly.
 
-Once the main and singleton packages have been deployed, they generally do not need to be deployed
-again. If a user were to remove the main or singleton package, the API can be used to reinstall them
-again.
+Once the Main and Singleton packages have been deployed, they generally do not need to be deployed
+again. If a user were to remove the Main or Singleton package, the API can be used to reinstall them
+again. If there is anything wrong with installed Main and Singleton packages, use Repair API to try
+and repair them.
 
 ```C#
-    if (DeploymentManager.GetStatus().Status != DeploymentStatus.Ok)
+    if (DeploymentManager.GetStatus().Status == DeploymentStatus.PackageInstallRequired)
```

Comment:
This does not do what the previous text recommends. Previous text says, "If there is anything wrong with installed Main and Singleton packages, use Repair API to try and repair them." But in this code, we don't call Repair for "anything wrong". In fact, we're not calling Repair at all!